apis/wikipedia.py

```diff
@@ -1,24 +1,159 @@
 from mediawiki import MediaWiki
+from bs4 import BeautifulSoup, SoupStrainer
 from typing import Dict
+from constants import wikipedia_api_url
+import requests
+from concurrent.futures import ThreadPoolExecutor
+
+
+def retrieve_short_description(data: str) -> str:
+
+    if 'wikibase-shortdesc' in data.keys():
+        return data['wikibase-shortdesc']
+
+    for _, value in list(data.items()):

```

Comment:
you can just got for `for value in data.values()`

```
In [3]: {'a': 'b'}.values()                 
Out[3]: dict_values(['b'])
```

Reply:
lol why didn't I think of that?! 🤦‍♂️ Will do 😅